consensus/byzantine_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"testing"
 	"time"
 
+	mdutils "github.com/ipfs/go-merkledag/test"
```

Comment:
Can we use another alias? mdutils sounds like markdown utils. Suggestion: dagutils?

Reply:
That's a default package name.

Reply:
I won't change that as that's actually temporary thing

Reply:
Later in block propagation PR instead of just Mock I will use Mock with Mock networking

Reply:
So I won't spend time for this renaming

Reply:
> That's a default package name.

But what does that even mean? Is the package is aliased like this in other contexts?